integrations/adding-an-integration/conneceting-to-k8s-cluster.md

```diff
@@ -0,0 +1,23 @@
+# Connecting to K8s cluster
+
+Connecting Aqueduct to a K8s cluster requires a `kubeconfig` file that allows a kubectl client to connect to the cluster.&#x20;
+
+Note that in order to use K8s, you must use
+* `AWS S3` or `GCS` as your metadata storage 
+*  an external data source (SQLlite is incompatible).&#x20;
```

Comment:
This line's a little confusing -- external might not be immediately obvious. Let's just say that workflows running on Kubernetes must have access to the data source for the workflow and will not work with the Aqueduct demo DB.